logstash-core-event-java/lib/logstash/string_interpolation.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+module LogStash
+  module StringInterpolation
+    extend self
+
+    # clear the global compiled templates cache
+    def clear_cache
+      Java::ComLogstash::StringInterpolation.get_instance.clear_cache;
```

Comment:
Style question - any reason you chose this syntax instead of `com.logstash.StringInterpolation` ?